### PR TITLE
Add instructions on how to migrate an existing project to using the template

### PR DESCRIPTION
**Pull request by grst**

### CODE REVIEW

File: {{cookiecutter.project_name}}/docs/template_usage.md [hunk 1]
```diff
@@ -419,6 +419,44 @@ The following hints may be useful to work with the template sync:
...
+You can also update existing projects to make use of this template to benefit from the latest-greatest
+tooling and automated template updates. This requires some manual work though. Here's one way how to do it
+
+1. Let's assume your repository is checked out to `$REPO`
+2. Clone your repository a second time to `${REPO}_cookiecutterized`
+3. Initialize an empty repository from this cookiecutter template:
+
+    ```bash
+    mkdir template && cd template
+    cruft create cruft create https://github.com/scverse/cookiecutter-scverse
```
Comment (flying-sheep) [inline {{cookiecutter.project_name}}/docs/template_usage.md:433]:
```suggestion
    cruft create https://github.com/scverse/cookiecutter-scverse
```

File: {{cookiecutter.project_name}}/docs/template_usage.md [hunk 2]
```diff
@@ -419,6 +419,44 @@ The following hints may be useful to work with the template sync:
...
+    ```
+
+4. remove everything from the existing repo
+
+    ```bash
+    cd ${REPO}_cookiecutterized
+    git co -b cookiecutterize
```
Comment (flying-sheep) [inline {{cookiecutter.project_name}}/docs/template_usage.md:440]:
```suggestion
    git switch -c cookiecutterize
```

File: {{cookiecutter.project_name}}/docs/template_usage.md [hunk 3]
```diff
@@ -419,6 +419,44 @@ The following hints may be useful to work with the template sync:
...
+    # move everything, including hidden folders, excluding `.git`.
+    rsync -av --exclude='.git' ./template/ ./${REPO}_cookiecutterized/
+    git add -A
+    git commit -m "init from template"
+    ```
+
+6. Migrate your project: Move over files from `$REPO` to `${REPO}_cookiecutterized`. Omit files that are not
+   needed anymore and manually merge files where required.
+
+7. Commit your changes. Merge the `cookiecutteriz` branch into the main branch, e.g. by making a pull request.
```
Comment (flying-sheep) [inline {{cookiecutter.project_name}}/docs/template_usage.md:458]:
```suggestion
7. Commit your changes. Merge the `cookiecutterize` branch into the main branch, e.g. by making a pull request.
```

**Review by flying-sheep — CHANGES_REQUESTED** [review]
Some minor things, otherwise looks good!

File: {{cookiecutter.project_name}}/docs/template_usage.md (hunk 2 above)
Comment (grst) [reply {{cookiecutter.project_name}}/docs/template_usage.md:440]: it's my personal alias. Thanks for spotting it!

**Review by flying-sheep — APPROVED** [review]

**Pull request merged**